python/ray/data/tests/test_parquet.py

```diff
@@ -756,6 +757,20 @@ def test_parquet_write(ray_start_regular_shared, fs, data_path, endpoint_url):
         fs.delete_dir(_unwrap_protocol(path))
 
 
+def test_parquet_partition_filter(ray_start_regular_shared, tmp_path):
+    empty_table = pa.table({})
+    pq.write_table(empty_table, tmp_path / "table.parquet")
+    # `spam` should be filtered out.
+    with open(tmp_path / "spam", "w"):
+        pass
+
+    ds = ray.data.read_parquet(
+        tmp_path, partition_filter=FileExtensionFilter("parquet")
+    )
+
+    assert ds.count() == 0
```

Comment:
sorry if I am wrong, but shouldn't `table.parquet` still there?

Reply:
ok, how about creating a non-empty table, so we can test out the filtering works, and not blindly drops all paths?